### ISSUE TITLE
Consider adress index usage (same as txi) 

### ISSUE THREAD

**Issue by thepiwo** [aeternity/ae_mdw#765]

**Comment by thepiwo** [comment]
maybe also relates to #725

**Comment by thepiwo** [comment]
@sborrazas do you still consider this helpful?

**Comment by sborrazas** [comment]
Yes, we can still look into doing this optimization for reducing DB size

**Comment by thepiwo** [comment]
@sborrazas I feel db size is not an issue currently, would it bring any other benefits?

**Comment by sborrazas** [comment]
> @sborrazas I feel db size is not an issue currently, would it bring any other benefits?

No, it wouldn't. Feel free to close it then
